Angular/src/app/login-dialog/login-dialog.component.ts

```diff
@@ -3,10 +3,11 @@ import { MatDialogRef } from "@angular/material/dialog";
 import { ApiService } from '../api.service';
 import { AuthService } from '../auth.service';
 import { HttpHeaders } from "@angular/common/http"
-import { first } from 'rxjs/operators';
+import { first, isEmpty } from 'rxjs/operators';
 import { stringify } from '@angular/compiler/src/util';
-import {Router} from '@angular/router';
+import { Router } from '@angular/router';
 import { ToastrService } from 'ngx-toastr';
+import { isNullOrUndefined } from '@swimlane/ngx-datatable';
```

Comment:
Implement yourself instead of adding another dept. Use below, it also checks if it's an empty string
`isNullOrUndefined(x) { return x === null || x === undefined || x.length === 0 }`